src/library/export/dlglibraryexport.cpp

```diff
@@ -233,7 +233,7 @@ void DlgLibraryExport::checkExistingDatabase() {
             m_pVersionCombo->clear();
             m_pVersionCombo->setEnabled(true);
             for (int versionIndex = 0;
-                    versionIndex < (int)e::all_versions.size();
+                    versionIndex < static_cast<int>(e::all_versions.size());
```

Comment:
Why not use std::size_t in the first place and remove the nerrowing cast here?

Reply:
because we add `-1` further down.